BlazarUI/app/stylus/vendor-overrides/bootstrap-overrides.styl

```diff
@@ -63,44 +117,58 @@ label
 // designs. This is not ideal because it causes some specificity issues
 // and we should settle on a consistent set of bootstrap overrides
 // across the entire app soon.
-.page-content--branch-state
-  a
-    color $color-calypso
+//.page-content--branch-state
+
+.btn
+  radius 3px
```

Comment:
Replaced a handful that I could. Had to keep the hover, focus, and active styles though since Bootstrap doesn't have props for those.